docs/en/sql-reference/data-types/lowcardinality.md

```diff
@@ -0,0 +1,57 @@
+---
+toc_priority: 51
+toc_title: LowCardinality
+---
+
+# LowCardinality Data Type {#lowcardinality-data-type}
+
+Changes the internal representation of other data types to be dictionary-encoded. 
+
+## Syntax {#lowcardinality-syntax}
+
+```sql
+LowCardinality(data_type)
+```
+
+**Parameters**
+
+- `data_type` — Supported ClickHouse [data type](index.md). `LowCardinality` is not efficient for some data types, see the [allow_suspicious_low_cardinality_types](../../operations/settings/settings.md#allow_suspicious_low_cardinality_types) setting description.
```

Comment:
Not all types are supported.
Only String, FixedString, Numbers (except decimals), Date, DateTime